pkg/storage/export/virt-exportserver/exportserver.go

```diff
@@ -120,6 +121,9 @@ func (er *execReader) Close() error {
 func (s *exportServer) initHandler() {
 	mux := http.NewServeMux()
 	for _, vi := range s.Paths.Volumes {
+		if hasPermissions := checkVolumePermissions(vi.Path); !hasPermissions {
+			golog.Fatalf("unable to manipulate %s's contents, exiting", vi.Path)
```

Comment:
This is a great addition and I think we should do the following to fully tighten up the logic:
1) add a readinessProbe to the exportserver container - can simply be a `/ready`http endpoint that returns "OK".  If the http server is running, we successfully passed the permission check
2) export controller will check the ready status of the container rather than the pod `status.Phase` when setting the export phase to `Running`

@alromeros do you want to do the above in this PR or subsequent one?

Reply:
I wouldn't mind doing it here but if this already fixes the bug I'd prefer to merge and backport ASAP. We can then handle the follow-up as a tech debt.

Reply:
Just added the readiness probe check to this PR @mhenriks